### ISSUE TITLE
Links in comments after editing in OnlyOffice lose their hrefs in MS Word

### ISSUE THREAD

**Issue by RuslanGabbasov** [ONLYOFFICE/DocumentServer#790]
**Do you want to request a *feature* or report a *bug*?**
*bug*

**What is the current behavior?**
Word 2019 before editing
![image](https://user-images.githubusercontent.com/20418709/75322387-70420380-5894-11ea-8392-3dfa79af9b11.png)

OnlyOffice editing comment 
![image](https://user-images.githubusercontent.com/20418709/75322103-fc076000-5893-11ea-89c5-7803a162880f.png)

Word 2019 after editing
![image](https://user-images.githubusercontent.com/20418709/75322137-088bb880-5894-11ea-9bee-0770d8a7410b.png)

**DocumentServer version:**
5.4.1.39

**Comment by Rita-Bubnova** [comment]
Hello, @RuslanGabbasov.
Thank you. I can confirm - this is a bug, issue 44640 in our internal issue tracker.

**Comment by RuslanGabbasov** [comment]
Additional bug in this story:
Rich text format in comments like text highlight background color loses after editing in OnlyOffice.

**Comment by Rita-Bubnova** [comment]
@RuslanGabbasov, You're right, this is the same issue 44640 in our internal issue tracker.
